lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -120,6 +119,11 @@ html.video-fullscreen {
         word-break: break-word;
     }
 
+    h3.unit-title {
+      margin-bottom: 0;
+      font-size: 1.5em;
+    }
+
```

Comment:
This file has a comment at the top that says:

> // NOTE: this file is deprecated, and we should not continue to add to this file. Use other partials as appropriate.

The changes to the `.studio-view` selector above are fine, but I think this new selector belongs somewhere else.